libsolidity/interface/FileReader.cpp

```diff
@@ -137,5 +134,17 @@ ReadCallback::Result FileReader::readFile(string const& _kind, string const& _so
 	}
 }
 
+string FileReader::toSourceUnitName(boost::filesystem::path const& _fsPath) const
+{
+	return _fsPath.generic_string();
+}
+
+boost::filesystem::path FileReader::fromSourceUnitName(string _sourceUnitName) const
+{
+	if (_sourceUnitName.find("file://") == 0)
+		_sourceUnitName.erase(0, 7);
+
+	return boost::filesystem::weakly_canonical(m_basePath / _sourceUnitName);
```

Comment:
Can `_sourceUnitName` be moved here? If not, why not make it a reference argument?

Reply:
It's getting modified so I'm basically using it as a substitute for a local variable. I could use a reference.